airbyte/types.py

```diff
@@ -24,6 +24,7 @@
     # We include them here for completeness.
     "object": sqlalchemy.types.JSON,
     "array": sqlalchemy.types.JSON,
+    "vector_array": sqlalchemy.types.ARRAY,
```

Comment:
vector_array is being interpreted as SQLAlchemy, which gets converted into Vector data type by TypeConverter.